integrations/standalone/tests/integration/other/callable-sub-process.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { test } from '@playwright/test';
+import { InscriptionView } from '../../pageobjects/InscriptionView';
+import { NameTestWithDisabledName, runTest } from '../parts';
+import { CreateProcessResult, createProcess } from '../../glsp-protocol';
+import { PermissionsTest } from '../parts/permissions';
+import { ProcessDataTest } from '../parts/process-data';

```

Comment:
Add these two tests to the parts index.ts file, then you can import them via '../parts'